localstack/services/awslambda/event_source_listeners/event_source_listener.py

```diff
@@ -0,0 +1,63 @@
+from typing import Any, Dict, Type
+
+from localstack.utils.objects import SubtypesInstanceManager
+from localstack.utils.threads import start_worker_thread
+
+
+class EventSourceListener(SubtypesInstanceManager):
+    INSTANCES: Dict[str, "EventSourceListener"] = {}
+
+    @staticmethod
+    def get_source_type() -> str:
```

Comment:
It seems like this method was renamed? There is another subclass in ext EventSourceListenerKafka which still uses the old signature.